docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -25,7 +25,7 @@ Playbook
 Command Line
 ============
 
-No notable changes
+* The location of the Galaxy token file has changed from ``~/.ansible_galaxy`` to ``~/.ansible/galaxy_token``, this can be configurable with the :ref:`galaxy_token_path` config.
```

Comment:
Is this two changes?
- The name of the file changes from `ansible_galaxy` to `galaxy_token`.
- The default location changes from `~/` to `~/.ansible/`. 
Are both configurable? In other words, can I set `galaxy_token_path=/my/path/to/my_token_file` or do I need to set `galaxy_token_path=/my/path/to/` and create a file called `/my/path/to/galaxy_token`?

Reply:
This porting guide entry is actually related to https://github.com/ansible/ansible/pull/59387, I added it here because I forgot to mention it in the review and I needed those changes in devel for this PR.

Before Ansible 2.9 there is a file `~/.ansible_galaxy` that contains only the line `token: my_token`. This is used to store the Galaxy token used for authentication. Since #59387 the location of this file has changed to `~/.ansible/galaxy_token` to better align where we put files. It is configurable with [GALAXY_TOKEN_PATH](https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1354) which is what the `:ref:...` line points to. Technically both the dir and filename has changed but `GALAXY_TOKEN_PATH` specifies the full path (not the dir) so if people want to be backwards compatible they need to do `ANSIBLE_GALAXY_TOKEN_PATH=~/.ansible_galaxy`.

Ultimately I don't believe this file is even really used as it's automatically set by the `ansible-galaxy login` command but I thought it prudent to have the porting guide entry in case people do.